### PR TITLE
fix missing handle 'update' event massage

### PR DESCRIPTION
**Pull request by fugr**
Signed-off-by: fugr <fugr2012@gmail.com>

### CODE REVIEW

File: cluster/engine.go [hunk 1]
```diff
@@ -1164,7 +1164,7 @@ func (e *Engine) handler(msg events.Message) error {
 			// These events refer to images so there's no need to update
 			// containers.
 			e.RefreshImages()
-		case "die", "kill", "oom", "pause", "start", "stop", "unpause", "rename":
+		case "die", "kill", "oom", "pause", "start", "stop", "unpause", "rename", "update":
```
Comment (allencloud) [inline cluster/engine.go]: I am afraid docker version below 1.10, there is no `update` event.

Comment (fugr) [inline cluster/engine.go]: you are right,'docker update' command added in after docker1.10

Comment (fugr) [inline cluster/engine.go]: @GordonTheTurtle how i could add other commit to this PR?

Comment (allencloud) [inline cluster/engine.go]: you can following:
1.modify the code in the same branch locally,
2.`git add`
3. when `git commit`, using `git commit -s --amend -m "xxx" `
4.`git push -f`

**Comment by fugr** [comment]
@GordonTheTurtle  thanks a lot!

**Comment by nishanttotla** [comment]
@fugr could you squash your commits into one commit please?

**Comment by vieux** [comment]
LGTM

**Comment by fugr** [comment]
@nishanttotla  how to do that

**Comment by allencloud** [comment]
you can try like:
http://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git

**Comment by fugr** [comment]
@nishanttotla   squash all commits into one commit,done

**Comment by fugr** [comment]
why tests failed?anyone knows?

**Comment by nishanttotla** [comment]
Test failures seem unrelated. LGTM

**Comment by dongluochen** [comment]
LGTM. Thanks @fugr.

**Pull request merged**